.circleci/config.yml

```
@@ -23,53 +14,40 @@ jobs:
       - checkout
       - restore_cache:
           key: dependency-cache-{{ checksum "package.json" }}
-      - run:
-          name: Install dependencies
-          command: npm i
+      - run: npm i
       - save_cache:
           key: dependency-cache-{{ checksum "package.json" }}
           paths:
             - ./node_modules
-  testnode6:
-    <<: *defaults
-    docker:
-      - image: circleci/node:6.11.5
-    steps:
-      - runtests
-  testnode8:
-    <<: *defaults
-    docker:
-      - image: circleci/node:8
-    steps:
-      - runtests
-  testnode10:
-    <<: *defaults
-    docker:
-      - image: circleci/node:10
-    steps:
-      - runtests
-  test:
+      - persist_to_workspace:
+          root: .
+          paths:
+            - ./node_modules
+  lint:
     <<: *defaults
     steps:
       - checkout
-      - restore_cache:
-          key: dependency-cache-{{ checksum "package.json" }}
-      - run:
-          name: Run tests
-          command: npm t
-  lint:
+      - attach_workspace:
+          at: .
+      - run: npm run lint -- --plugin log
+  test:
+    parameters:
+      image:
+        description: "node version"
+        default: "lts"
```

You're using complete image names in "image" parameter
```suggestion
        default: "cimg/node:lts"
```